Services/include/services.h

```diff
@@ -52,6 +52,8 @@
 #define TC_GENERAL_SERVICE 11
 // UPDATER_SERVICE
 #define TC_UPDATER_SERVICE 12
+// ADCS SERVICE
+#define TC_ADCS_SERVICE 14
```

Comment:
Port 14 is taken. Check [system.py](https://github.com/AlbertaSat/ex2_ground_station_software/blob/master/src/groundStation/system.py) to find a free one.

Reply:
looks like 17 is free. should he use that one?

Reply:
isn't housekeeping on 17? There's nothing on 13

Reply:
Oh housekeeping was on 9 but now that you mention it, I remember seeing housekeeping was moved to 17. 13 was free so I put the logger on that one

Reply:
Sure I can put adcs on 17

Reply:
No you can't. housekeeping is on it. but it looks like 18 is free unless someone says otherwise